src/lara-plugin/events/index.spec.ts

```diff
@@ -27,4 +27,17 @@ describe("Events helper", () => {
       expect(handler).toHaveBeenNthCalledWith(1, e);
     });
   });
+
+  describe("InteractiveSupportedFeatures event", () => {
+    it("provides working API for event handling", () => {
+      const handler = jest.fn();
+      events.onInteractiveSupportedFeatures(handler);
+      const e: IInteractiveSupportedFeaturesEvent = { container: document.createElement("div"), supportedFeatures: {} };
+      events.emitInteractiveSupportedFeatures(e);
+      expect(handler).toHaveBeenNthCalledWith(1, e);
+      events.offInteractiveSupportedFeatures(handler);
+      events.emitInteractiveSupportedFeatures(e);
+      expect(handler).toHaveBeenNthCalledWith(1, e);
+    });
+  });
```

Comment:
As in the previous PR, does @scytacki still want us to wrap all additions/subtractions in the `lara-plugin` folder with comments or at some point is it just going to make the code hard to read?

Reply:
I just did a round of synchronization so that changes in this PR were added to the corresponding LARA PR and vice-versa to the extent possible. All remaining changes (mostly import statements) are so marked.

Reply:
Thanks Kirk. If you are able to drop the jquery from lara-plugin, then the code will probably diverge quite a bit. At that point it might sense to drop the commenting approach.  Or perhaps if that code can be sufficiently isolated (both in LARA and here), then we could consider keeping them.

If we do make it over that jquery hurdle and there is still a significant amount of shared code, we might be in a good place to estimate what it would take to turn the shared code into a module in the LARA code base that is imported in the Activity Player.  I only say 'estimate' because I'm not convinced it is worth doing.